### PR TITLE
ref/runner: clarify AWS options

### PR DESCRIPTION
**Pull request by casperdcl**
- related https://github.com/iterative/cml/issues/868

### CODE REVIEW

File: content/docs/ref/runner.md [hunk 1]
```diff
@@ -52,9 +52,10 @@ Any [generic option](/doc/ref) in addition to:
   purposes.
 - `--cloud-aws-security-group=<...>`:
   [AWS security group](https://docs.aws.amazon.com/vpc/latest/userguide/VPC_SecurityGroups.html)
-  name.
-- `--cloud-aws-subnet-id=<...>`:
-  [AWS subnet](https://docs.aws.amazon.com/vpc/latest/userguide/VPC_Subnets.html#subnet-basics).
+  name or ID.
```
Comment (casperdcl) [inline content/docs/ref/runner.md:55]: is this correct order of precedence?

File: content/docs/ref/runner.md [hunk 2]
```diff
@@ -52,9 +52,10 @@ Any [generic option](/doc/ref) in addition to:
...
+- `--cloud-aws-subnet=<...>`:
+  [AWS subnet](https://docs.aws.amazon.com/vpc/latest/userguide/VPC_Subnets.html#subnet-basics)
+  name or ID.
```
Comment (0x2b3bfa0) [inline content/docs/ref/runner.md:58]: As [stated](https://iterativeai.slack.com/archives/C01900GSB4J/p1642078655064100?thread_ts=1642036384.053000&cid=C01900GSB4J), subnets don't [always] have unique names.
```suggestion
  name.
```

File: content/docs/ref/runner.md (hunk 1 above)
Comment (0x2b3bfa0) [inline content/docs/ref/runner.md:55]:
```suggestion
  identifier.
```

While security group names asre also unique, I would prefer to deprecate that option and encourage users to use identifiers instead.

Comment (0x2b3bfa0) [reply content/docs/ref/runner.md:55]: [Quite the opposite](https://github.com/iterative/terraform-provider-iterative/blob/e50ea7911a88de4229e1bb16c19fe8dd94aac48b/iterative/aws/provider.go#L182), but doesn't matter if we drop support for names, at least in the documentation. Moved to https://github.com/iterative/cml.dev/pull/185#discussion_r786990775.

File: content/docs/ref/runner.md (hunk 2 above)
[the remark this answers is not in the document]
Comment (0x2b3bfa0) [reply content/docs/ref/runner.md:58]: [Quite the opposite](https://github.com/iterative/terraform-provider-iterative/blob/e50ea7911a88de4229e1bb16c19fe8dd94aac48b/iterative/aws/provider.go#L182), but doesn't matter if we drop support for names, at least in the documentation. Moved to https://github.com/iterative/cml.dev/pull/185#discussion_r786992411.

**Review by 0x2b3bfa0 — CHANGES_REQUESTED** [review]
Do we have any good reason for encouraging the use of resource names?

**Review by 0x2b3bfa0 — APPROVED** [review]
🙏🏼

**Pull request merged**